### ISSUE TITLE
markdownlint(MD025) triggered by metadata

### ISSUE THREAD

**Issue by linonetwo** [markdownlint/markdownlint#257]
```md
---
layout: post
title: '用浏览器输入框代替Alfred'
---
# 用浏览器输入框代替 Alfred
```
This will trigger MD025

While this not:

```md
---
layout: post
---
# 用浏览器输入框代替 Alfred
```

**Comment by DavidAnson** [comment]
I’m guessing you are using the Node.js version? If so, this is new behavior and correct. It can be configured. More info here: https://github.com/DavidAnson/markdownlint/issues/177

**Comment by linonetwo** [comment]
Oh, yes! You are right, this is actually the correct behavior.

I'm just used to the old behavior and confused...
